Goal: fix dist_conf/debian

Author: Mathieu Parent <math.parent@gmail.com>

Upstream status: Already included, will be in 2.2.1

Index: b/dist_conf/debian
===================================================================
--- a/dist_conf/debian
+++ b/dist_conf/debian
@@ -1,20 +1,21 @@
+BASHELL=/bin/bash
+PERL=${bindir}/perl
+PHP=${bindir}/php
+KOLABRC=${sbindir}/kolab
+#RPM=${bindir}/rpm
+TAR=/bin/tar
+
 kolab_usr=kolab
 kolab_grp=kolab
 kolab_musr=kolab
 kolab_mgrp=kolab
 kolab_uid=1002
 kolab_gid=1002
-restricted_kolab_usr=kolab
-restricted_kolab_grp=kolab
 kolab_rusr=${restricted_kolab_usr}
 kolab_rgrp=${restricted_kolab_grp}
 kolab_susr=root
-pki_grp=${kolab_rgrp}
-
-BASHELL=/bin/bash
-PERL=${bindir}/perl
-PHP=${bindir}/php
-KOLABRC=${sbindir}/kolab
+restricted_kolab_usr=kolab
+restricted_kolab_grp=kolab
 
 kolab_rcdir=${sysconfdir}/init.d
 pkgdocdir=${datadir}/doc/kolab
@@ -35,8 +36,6 @@
 
 fsl_logfile=${localstatedir}/log/fsl
 
-sysrundir=${localstatedir}/run
-
 clamav_confdir=${sysconfdir}/clamav
 clamav_socket=${localstatedir}/clamav/clamd.sock
 clamav_logfile=${localstatedir}/log/clamav/clamav.log
@@ -57,10 +56,13 @@
 emailserver_localstatedir=${localstatedir}/spool/postfix
 emailserver_logfile=${localstatedir}/log/mail.log
 emailserver_usr=root
-emailserver_tables_usr=root
 emailserver_musr=kolab
 emailserver_grp=root
+#emailserver_mail_owner=
+#emailserver_setgid_grp=
+#emailserver_default_privs=
 emailserver_master_usr=kolab
+emailserver_tables_usr=root
 
 imap_confdir=${sysconfdir}
 imap_confperm=0640
@@ -74,13 +76,18 @@
 imap_lmtp=${localstatedir}/run/cyrus/socket/lmtp
 imap_notify_socket=${localstatedir}/run/cyrus/socket/notify
 
+
 kolab_statedir=${localstatedir}/lib/kolab
 kolab_logdir=${localstatedir}/log/kolab
 kolab_pidfile=${sysrundir}/kolab.pid
 kolab_mailboxuiddb=${kolab_statedir}/mailbox-uidcache.db
+kolab_smtpdpolicy_conffile_usr=${kolab_musr}
+kolab_smtpdpolicy_conffile_grp=${kolab_grp}
+#kolab_php_module_prefix=
 kolab_scriptsdir=${sbindir}
 kolab_php_smarty_prefix=smarty/libs
 smarty_compiledir=/var/cache/kolab-webadmin/templates
+
 # Kolab web user interface
 kolab_wui=/admin
 
@@ -89,15 +96,18 @@
 ldapserver_replogfile=${ldapserver_statedir}/replog
 ldapserver_argsfile=${sysrundir}/slapd/slapd.args
 ldapserver_logfile=${localstatedir}/log/syslog
+#ldapserver_rundir=
 ldapserver_pidfile=${sysrundir}/slapd/slapd.pid
 ldapserverslurpd_pidfile=${sysrundir}/slapd/slurpd.pid
 ldapserver_schemadir=${sysconfdir}/ldap/schema
 ldapserver_confdir=${sysconfdir}/ldap
 ldapserver_usr=openldap
 ldapserver_grp=openldap
+ldapserver_rgrp=openldap
 ldapserver_rusr=openldap
 
 webserver_document_root=/usr/share/kolab
+#webserver_tmpdir=
 webserver_cgidir=${webserver_document_root}/cgi-bin
 webserver_sessions=${localstatedir}/cache/kolab/httpd_sessions
 webserver_logdir=${localstatedir}/log/apache2
@@ -125,17 +135,20 @@
 sasl_smtpconffile=${sysconfdir}/postfix/sasl/smtpd.conf
 sasl_logfile=${localstatedir}/log/saslauthd.log
 
-resmgr_logfile=${localstatedir}/log/resmgr.log
+sysrundir=${localstatedir}/run
+
+resmgr_logdir=${localstatedir}/log
 resmgr_confdir=${sysconfdir}/kolab
-resmgr_filterdir=${localstatedir}/lib/kolab/resmgr/filter
-resmgr_scriptsdir=/usr/share/kolab/filter
 resmgr_conffile_usr=kolab
 resmgr_conffile_grp=root
+#resmgr_tmpdir=
+resmgr_scriptsdir=/usr/share/kolab/filter
 resmgr_usr=root
 resmgr_grp=www-data
+resmgr_filterdir=${localstatedir}/lib/kolab/resmgr/filter
 
 fbview_logfile=${localstatedir}/resmgr/fbview.log
-freebusy_logfile=${localstatedir}/resmgr/freebusy.log
+freebusy_logdir=${localstatedir}/log
 freebusy_confdir=${sysconfdir}/kolab
 freebusy_cachedir=${localstatedir}/lib/kolab/freebusy-cache
 freebusy_usr=root
@@ -146,8 +159,6 @@
 
 backupdir=${localstatedir}/backups/kolab
 
-kolab_smtpdpolicy_conffile_usr=${kolab_musr}
-kolab_smtpdpolicy_conffile_grp=${kolab_grp}
+pki_grp=${kolab_rgrp}
 
 aliases_file=${sysconfdir}/aliases
-
